### PR TITLE
[2.5] update rancher-k3s-upgrader chart version 0.2.1

### PR DESCRIPTION
**Pull request by MonzElmasry**
This pr updates the version of rancher-k3s-upgrader chart
New version includes a keep policy for the cattle-system namespace so it doest get deleted when chart is uninstalled

2.4 pr : https://github.com/rancher/system-charts/pull/353

### CODE REVIEW

**Review by rmweir — APPROVED** [review]
LGTM

**Comment by davidnuzik** [comment]
Unlike the 2.4 PR (https://github.com/rancher/system-charts/pull/353) we can merge this one, after we get second LGTM.

**Review by dramich — APPROVED** [review]

**Pull request merged**
